src/Bridge/Doctrine/Orm/Util/IdentifierManagerTrait.php

```diff
@@ -86,4 +88,24 @@ private function normalizeIdentifiers($id, ObjectManager $manager, string $resou
 
         return $identifiers;
     }
+
+    /**
+     * Check if attributes contains convert to false.
+     *
+     * @param array $attributes
+     *
+     * @return bool
+     */
+    private function shouldConvert(array $attributes = null): bool
+    {
+        foreach ((array) $attributes as $attribute) {
```

Comment:
it seems to me, If attributes will be null then invalid argument supplied to foreach will be thrown